pom.xml

```diff
@@ -1080,6 +1080,7 @@ under the License.
           <artifactId>maven-release-plugin</artifactId>
           <configuration>
             <autoVersionSubmodules>true</autoVersionSubmodules>
+            <releaseProfiles>apache-release,reporting</releaseProfiles>
```

Comment:
Should be for next PR - it is out of scope for me

Reply:
Created https://issues.apache.org/jira/browse/MPOM-358 and removed from this PR.